### PR TITLE
Add new IOU fields to OneToN description

### PR DESCRIPTION
**Pull request by karlb**
Relates to https://github.com/raiden-network/raiden-services/issues/267
and https://github.com/raiden-network/raiden-contracts/issues/849.

### CODE REVIEW

File: service_contracts.rst [hunk 1]
```diff
@@ -150,9 +154,12 @@ expiration_block).
 .. [#sig] The signature is calculated by
           ::
 
-               ecdsa_recoverable(privkey,
-                                 sha3_keccak("\x19Ethereum Signed Message:\n104"
-                                             || sender || receiver || amount || expiration_block ))
+               ecdsa_recoverable(
+                   privkey,
+                   sha3_keccak("\x19Ethereum Signed Message:\n104"
```
Comment (palango) [inline service_contracts.rst]: The size needs to be adapted here.

Comment (karlb) [reply service_contracts.rst]: Good catch! I'll fix it.

Comment (pirapira) [reply service_contracts.rst]: Did it happen in the implementation?  Looking...  :eyes:

Comment (pirapira) [reply service_contracts.rst]: Yes, it did happen there.

**Pull request merged**